### PR TITLE
isEqualTo Optimization

### PR DESCRIPTION
**Pull request by tkcjesse**
Replaced a bunch of == for constant stuff and things that should be
good. This needs to be tested or at least scrolled threw by multiple people.. Obviously testing each one of these would be extremely time consuming... If someone has a few to check it out
that'd be awesome.

### CODE REVIEW

File: Altis_Life.Altis/core/actions/fn_restrainAction.sqf [hunk 1]
```diff
@@ -11,8 +11,8 @@ _unit = cursorObject;
 if (isNull _unit) exitWith {}; //Not valid
 if (player distance _unit > 3) exitWith {};
 if (_unit getVariable "restrained") exitWith {};
-if (side _unit == west) exitWith {};
-if (player == _unit) exitWith {};
+if (side _unit isEqualTo west) exitWith {};
+if (player player isEqualTo _unit) exitWith {};
```
Comment (sargken) [inline Altis_Life.Altis/core/actions/fn_restrainAction.sqf]: dual player?

Comment (tkcjesse) [inline Altis_Life.Altis/core/actions/fn_restrainAction.sqf]: Yeah I'll get that. theres 4 more like this.

**Comment by Jawshy** [comment]
Tested with no issues found for: briefing, fn_catchFish, fn_dropFishingNet, fn_impoundAction, fn_mine, fn_pickupItem, fn_doorAnimate, fn_vehInvSearch, fn_deviceMine, fn_addContainer, and fn_addHouse.

File: Altis_Life.Altis/core/cop/fn_radar.sqf [hunk 2]
```diff
@@ -10,9 +10,8 @@ private ["_speed","_vehicle"];
 _vehicle = cursorObject;
 _speed = round speed _vehicle;
 
-if ((_vehicle isKindOf "Car") && (currentWeapon player) == "hgun_P07_snds_F") then {
-    switch (true) do
-    {
+if ((_vehicle isKindOf "Car") && (currentWeapon player) isEqualTo "hgun_P07_snds_F") then {
```
Comment (BoGuu) [inline Altis_Life.Altis/core/cop/fn_radar.sqf]: Marking for future reference

File: Altis_Life.Altis/core/functions/fn_handleItem.sqf [hunk 3]
```diff
@@ -407,7 +407,7 @@ if (_bool) then {
                     default {_ispack = true;};
                 };
 
-                if (_item == "MineDetector") then {
+                if (_item isEqualTo "MineDetector") then {
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_handleItem.sqf]: Marking for future reference

File: Altis_Life.Altis/core/functions/fn_nearUnits.sqf [hunk 4]
```diff
@@ -16,7 +16,7 @@ _radius = [_this,2,30,[0]] call BIS_fnc_param;
 _ret = false;
 
 //Error check
-if (_faction == sideUnknown) exitWith {_ret};
+if (_faction isEqualTo sideUnknown) exitWith {_ret};
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_nearUnits.sqf]: Marking for future reference

File: Altis_Life.Altis/core/functions/fn_playerSkins.sqf [hunk 5]
```diff
@@ -11,32 +11,32 @@ private["_skinName"];
 switch (playerSide) do {
     case civilian: {
         if (LIFE_SETTINGS(getNumber,"civ_skins") isEqualTo 1) then {
-            if (uniform player == "U_C_Poloshirt_blue") then {
+            if (uniform player isEqualTo "U_C_Poloshirt_blue") then {
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_playerSkins.sqf]: Marking for future reference

File: life_hc/MySQL/General/fn_updatePartial.sqf [hunk 6]
```diff
@@ -13,7 +13,7 @@ _uid = [_this,0,"",[""]] call BIS_fnc_param;
 _side = [_this,1,sideUnknown,[civilian]] call BIS_fnc_param;
 _mode = [_this,3,-1,[0]] call BIS_fnc_param;
 
-if (_uid == "" || _side == sideUnknown) exitWith {}; //Bad.
+if (_uid isEqualTo "" || _side isEqualTo sideUnknown) exitWith {}; //Bad.
```
Comment (BoGuu) [inline life_hc/MySQL/General/fn_updatePartial.sqf]: Marking for future reference

File: life_hc/MySQL/Vehicles/fn_keyManagement.sqf [hunk 7]
```diff
@@ -12,7 +12,7 @@ _uid = [_this,0,"",[""]] call BIS_fnc_param;
 _side = [_this,1,sideUnknown,[sideUnknown]] call BIS_fnc_param;
 _mode = [_this,3,0,[0]] call BIS_fnc_param;
 
-if (_uid == "" || _side == sideUnknown) exitWith {}; //BAAAAAAAAADDDDDDDD
+if (_uid isEqualTo "" || _side isEqualTo sideUnknown) exitWith {}; //BAAAAAAAAADDDDDDDD
```
Comment (BoGuu) [inline life_hc/MySQL/Vehicles/fn_keyManagement.sqf]: Marking for future reference

File: life_hc/MySQL/Vehicles/fn_getVehicles.sqf [hunk 8]
```diff
@@ -15,7 +15,7 @@ _type = [_this,2,"",[""]] call BIS_fnc_param;
 _unit = [_this,3,ObjNull,[ObjNull]] call BIS_fnc_param;
 
 //Error checks
-if (_pid == "" || _side == sideUnknown || _type == "" || isNull _unit) exitWith {
+if (_pid isEqualTo "" || _side isEqualTo sideUnknown || _type isEqualTo "" || isNull _unit) exitWith {
```
Comment (BoGuu) [inline life_hc/MySQL/Vehicles/fn_getVehicles.sqf]: Marking for future reference

File: life_hc/MySQL/Vehicles/fn_vehicleCreate.sqf [hunk 9]
```diff
@@ -14,7 +14,7 @@ _vehicle = [_this,2,ObjNull,[ObjNull]] call BIS_fnc_param;
 _color = [_this,3,-1,[0]] call BIS_fnc_param;
 
 //Error checks
-if (_uid == "" || _side == sideUnknown || isNull _vehicle) exitWith {};
+if (_uid isEqualTo "" || _side isEqualTo sideUnknown || isNull _vehicle) exitWith {};
```
Comment (BoGuu) [inline life_hc/MySQL/Vehicles/fn_vehicleCreate.sqf]: Marking for future reference

File: life_server/Functions/MySQL/fn_updatePartial.sqf [hunk 10]
```diff
@@ -11,7 +11,7 @@ _uid = [_this,0,"",[""]] call BIS_fnc_param;
 _side = [_this,1,sideUnknown,[civilian]] call BIS_fnc_param;
 _mode = [_this,3,-1,[0]] call BIS_fnc_param;
 
-if (_uid == "" || _side == sideUnknown) exitWith {}; //Bad.
+if (_uid isEqualTo "" || _side isEqualTo sideUnknown) exitWith {}; //Bad.
```
Comment (BoGuu) [inline life_server/Functions/MySQL/fn_updatePartial.sqf]: Marking for future reference

File: life_server/Functions/Systems/fn_vehicleCreate.sqf [hunk 11]
```diff
@@ -12,7 +12,7 @@ _vehicle = [_this,2,ObjNull,[ObjNull]] call BIS_fnc_param;
 _color = [_this,3,-1,[0]] call BIS_fnc_param;
 
 //Error checks
-if (_uid == "" || _side == sideUnknown || isNull _vehicle) exitWith {};
+if (_uid isEqualTo "" || _side isEqualTo sideUnknown || isNull _vehicle) exitWith {};
```
Comment (BoGuu) [inline life_server/Functions/Systems/fn_vehicleCreate.sqf]: Marking for future reference

**Pull request merged**